iindx.css

```diff
@@ -0,0 +1,73 @@
+body {
+  margin: 0;
+  background-color: rgba(0, 0, 0, 0.05);
+}
+
+.flexrow {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 20px 0;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 20px;
+}
+
+.conterwidth {
+  width: 80%;
+  margin: 20px auto;
+  box-shadow: 0 8px 6px #aaa;
+  border-radius: 5px;
+}
+
```

Comment:
please add more descriptive names to your classes kindklly go back and change it.